### DISCUSSION TITLE
Trim line trailing white spaces on file save

### DISCUSSION THREAD

**Discussion by fvacek** [helix-editor/helix#3557] [Ideas]
It would be nice to have editor config option to set Helix to trim line trailing white-spaces on file save.

**Comment by DannyJJK** [comment]
I am missing this too. I think it was also discussed in this issue: https://github.com/helix-editor/helix/issues/1481

For now, you could use a formatter in your `languages.toml` file
https://docs.helix-editor.com/languages.html
